app.rb

```diff
@@ -42,9 +42,10 @@ def create_person
     print 'Age: '
     age = gets.chomp
     if res == 1
-      person = Student.new(age, nil, name)
-      puts "Student created successfully. ID: #{person.id}"
-      @people << person
+      print 'Has parent permission? [Y/N]: '
+      parent_permission = gets.chomp.downcase == 'y'
+      person = Student.new(age, nil, name, parent_permission: parent_permission)
```

Comment:
- [ ] Great work on issues stated by the previous reviewer but while you fix it there is another issue raised. When I try to create a student and after that list all people the students are not shown in the console, kindly fix this issue